nevergrad/functions/images/imagelosses.py

```diff
@@ -7,7 +7,12 @@
 import typing as tp
 import torch
 import numpy as np
-import imquality.brisque as brisque
+
+try:
+    import imquality.brisque as brisque
+except ImportError:
+    print("Brisque will fail: imquality missing")
+
```

Comment:
please move this to the function used and raise ExperimentNotSupportedError

Reply:
Done.